rbkit-lib/cpu/cpumapping.h

```diff
@@ -0,0 +1,28 @@
+#include <QString>
+
+#ifndef MAPPING
+#define MAPPING
+
+inline namespace MAPS {
+    enum Map {
+        event_type,
```

Comment:
also, coding conventions are not followed, ie we should use camelCase since we are coding in Qt